README.md

```diff
@@ -1,8 +1,22 @@
 # CMake Modules
 
 This repository contains common CMake modules and a collection of find scripts
-to locate non-CMake dependencies. To use it, create a .gitexternals file in your
-project with the content:
+to locate non-CMake dependencies. The recommended way to use it is:
```

Comment:
As a matter of fact, the only recommended way I would document is the new submodule one. I was considering after we have updated all our projects, we can remove GitExternal.cmake and move what we still need for subproject cloning to SubProject.cmake. If we don't reach a conclusion now, we could discuss this in the next team-meeting.